packages/graphile-build-pg/src/plugins/PgColumnDescriptionsPlugin.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import type { Plugin } from "graphile-build";
+import { parseTags } from "../utils";
+
+export default (function PgColumnDescriptionsPlugin(builder) {
+  builder.hook("GraphQLObjectType:fields:field", (field, build, context) => {
+    const {
+      scope: {
+        isPgRowType,
+        isPgCompoundType,
+        pgIntrospection: table,
+        pgFieldIntrospection,
+      },
+    } = context;
+    if (
+      !(isPgRowType || isPgCompoundType) ||
+      !table ||
+      table.kind !== "class" ||
+      !pgFieldIntrospection ||
+      typeof pgFieldIntrospection.description !== "string"
+    ) {
+      return field;
+    }
+    const parsed = parseTags(pgFieldIntrospection.description);
```

Comment:
So I've been thinking about this, and I think we should move the `parseTags` call into `PgIntrospectionQuery` - then other plugins can make use of the other tags also without having to re-parse the tags. We can also implement this for all the other types (functions, tables, etc) so that they don't have tags included in their `description` fields.

Reply:
What do you think?

Reply:
\*PgIntrospectionPlugin

Reply:
I agree.  The only downside is tighter coupling - we won't have an easy way to drop the tagging feature from the plugin stack.  But if you're willing to support this as a first-class feature in the core, I think the pros (avoiding re-parsing, including them on other types, and simpler development of tag-based plugins) outweigh that con.

Reply:
Good point; let's add "pgEnableTags" as an option and have it default true in `postgraphile-core` - that way people who don't want it can turn it off (and we'd just set `tags` to be `[]` on the relevant introspection fields).

Reply:
(I do want tags in core, they're going to be super useful for things that can't be done automatically 👍 )